Client/Frontend/Home/TopSites/DataManagement/TopSitesDataAdaptor.swift

```diff
@@ -103,6 +107,9 @@ class TopSitesDataAdaptorImplementation: TopSitesDataAdaptor, FeatureFlaggable,
 
         sites.removeDuplicates()
 
+        // This will prevent any Sponsored tile from appearing if its SLD matches the default search engine.
+        sites = topSitesDataUtility.removeSiteMatching(defaultSearchEngine, from: sites)
```

Comment:
If we renamed this `removeSiteMatchingSites(in: defaultSearchEngine, from: sites)`, it would be clearer, and then no longer need a comment.